### ISSUE TITLE
wildcard certs don't work on Windows

### ISSUE THREAD

**Issue by Strandedpirate** [FiloSottile/mkcert#232]
Windows 10 x64
Chrome 79.0.3945.88

Wildcard certificates created per the doc do not work on Windows 10. The cert shows up in Chrome as Invalid for whoami.localhost.

`mkcert -key-file key.pem -cert-file cert.pem *.localhost`

Non-wildcard certificates with specific subject alternative names do work. This shows up as Valid in Chrome for whoami.localhost.

`mkcert -key-file key.pem -cert-file cert.pem blog.example.com whoami.localhost`

**Comment by blotsome** [comment]
I may have this problem as well. I used wildcard certs and both Chrome and Firefox in windows don't show the green lock, Chrome says cert is invalid, and Firefox "connection is not secure".

**Comment by blotsome** [comment]
Assuming this is a Windows issue, is there any work around? Or say I am setting up half a dozen subdomains for various selfhosted services, plex.server.lan, bw.server.lan, pihole.server.lan, is there a more automated way to create and import multiple certs instead of doing them one at a time?

**Comment by timwis** [comment]
Have you tried surrounding the wildcard hostname in quotes like the documentation example? e.g. `mkcert "*.localhost"`

Alternatively, I read somewhere that Windows doesn't recognise `pem` files as certificates. Try using `-cert-file sitename.crt -key-file sitename.key`.

Note that I'm just another mkcert using making a guess. Haven't tried it on windows.

**Comment by blotsome** [comment]
I tried creating a single domain cert, and importing the CA again using mkcert, and didn't seem to do anything, so I then manually loaded the cert into Chrome/windows database, and still nothing. I rebooted Chrome and it worked. But I also noticed I was getting the lock ICON on my wildcard cert domains... so I updated NGINX to only use the wildcard cert again, restarted NGINX, restarted Chrome, and they appear to be working for now, in both Chrome and Firefox. I don't know if it was restarting Chrome (which I'm sure I did the first time around) or manually loading the certs or maybe just the first import failed for some reason? I can't explain it, but for now my 5 subdomains are up and running with a single wildcard cert (though pihole still isn't showing the lock icon for some reason). I wish I had a better explanation.
